perception/traffic_light_selector/README.md

```diff
@@ -0,0 +1,9 @@
+# traffic_light_selector
+
+## selector node
+
+A node that merges traffic light data from traffic light recognition and V2X to provide a planning component.
+
+## converter node
+
+A temporary node that converts old message (autoware_auto_perception_msgs) to new message (autoware_perception_msgs).
```

Comment:
If possible, can you write it in the same format as the other packages?
Example : https://github.com/autowarefoundation/autoware.universe/tree/main/perception/lidar_apollo_instance_segmentation

Reply:
Update readme. https://github.com/autowarefoundation/autoware.universe/pull/2903/commits/b95cc297bd77fc3d159e67ed801dbcdf7c0a03be